app/src/main/java/com/endava/myendava/activities/SignInActivity.java

```diff
@@ -66,17 +66,10 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_sign_in);
         mUnbinder = ButterKnife.bind(this);
-        setTransparentStatusBar();
         setupModule();
+        hideProgressBar(mProgressBar);
```

Comment:
Make the initial state of the progress bar 'Gone' from the xml instead of calling this method on initialization of the activity